website/api/_cython/_tokenc.jade

```diff
@@ -0,0 +1,268 @@
+//- 💫 DOCS > API > CYTHON > STRUCTS > TOKENC
+
+p
+    |  Cython data container for the #[code Token] object.
+
++aside-code("Example").
+    token = &doc.c[3]
+    token_ptr = &doc.c[3]
+
++table(["Name", "Type", "Description"])
+    +row
+        +cell #[code lex]
+        +cell #[code const LexemeC*]
+        +cell A pointer to the lexeme for the token.
+
+    +row
+        +cell #[code morph]
+        +cell #[code uint64_t]
+        +cell An ID allowing lookup of morphological attributes.
+
+    +row
+        +cell #[code pos]
+        +cell #[code univ_pos_t]
+        +cell Coarse-grained part-of-speech tag.
+
+    +row
+        +cell #[code spacy]
+        +cell #[code bint]
+        +cell A binary value indicating whether the token has trailing whitespace.
+
+    +row
+        +cell #[code tag]
+        +cell #[+abbr("uint64_t") #[code attr_t]]
+        +cell Fine-grained part-of-speech tag.
+
+    +row
+        +cell #[code idx]
+        +cell #[code int]
+        +cell The character offset of the token within the parent document.
+
+    +row
+        +cell #[code lemma]
+        +cell #[+abbr("uint64_t") #[code attr_t]]
+        +cell Base form of the token, with no inflectional suffixes.
+
+    +row
+        +cell #[code sense]
+        +cell #[+abbr("uint64_t") #[code attr_t]]
+        +cell Space for storing a word sense ID, currently unused.
+
+    +row
+        +cell #[code head]
+        +cell #[code int]
+        +cell Offset of the syntactic parent relative to the token.
+
+    +row
+        +cell #[code dep]
+        +cell #[+abbr("uint64_t") #[code attr_t]]
+        +cell Syntactic dependency relation.
+
+    +row
+        +cell #[code l_kids]
+        +cell #[code uint32_t]
+        +cell Number of left children.
+
+    +row
+        +cell #[code r_kids]
+        +cell #[code uint32_t]
+        +cell Number of right children.
+
+    +row
+        +cell #[code l_edge]
+        +cell #[code uint32_t]
+        +cell Offset of the leftmost token of this token's syntactic descendents.
+
+    +row
+        +cell #[code r_edge]
+        +cell #[code uint32_t]
+        +cell Offset of the rightmost token of this token's syntactic descendents.
+
+    +row
+        +cell #[code sent_start]
+        +cell #[code int]
+        +cell
+            |  Ternary value indicating whether the token is the first word of
+            |  a sentence. #[code 0] indicates a missing value, #[code -1]
+            |  indicates #[code False] and #[code 1] indicates #[code True].
```

Comment:
I am not 100% sure about this one.
Usually in my experiments, sent_start is 0 for every token that is not at the beginning of a sentence and 1 for each token at the beginning of a sentence **with the exception of the first token of the doc which as it at 0 also.**

Reply:
I think you're right -- will amend that, thanks.